external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -89,6 +89,10 @@ public static void main (String args[]) {
                     printUsageAndExit(options, OPTION_ZK_SERVERS_LONG + " and " + OPTION_ZK_COMMITTED_NODE_LONG + " are required  with " +
                             OPTION_OLD_CONSUMER_LONG);
                 }
+                String zkNode = commandLine.getOptionValue(OPTION_ZK_COMMITTED_NODE_LONG);
```

Comment:
While adding OPTION_ZK_NODE_COMMITTED_NODE_LONG to options it is added using the method options.addOption(String opt, String longOpt, boolean hasArg, String description) The third argument is set to true in the call which specifies that the argument must have a value. Does not that take care of null or empty value? Do we need to have this explicit check here?